.github/workflows/test.yml

```diff
@@ -199,7 +199,7 @@ jobs:
           report_individual_runs: true
 
   coverage:
-    if: ${{ github.event.pull_request.user.login != 'dependabot[bot]' && github.event_name != 'release' }}
+    if: ${{ github.event.pull_request.user.login != 'dependabot[bot]' && github.workflow != 'Release' }}
```

Comment:
I guess the workflow will reflect the initially triggered workflow.

Reply:
yes true. The caller of this workflow